### PR TITLE
Add support for argument lists

### PR DESCRIPTION
**Pull request by nex3**

### CODE REVIEW

File: lib/src/protofier.dart [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+        ..quoted = value.hasQuotes;
+    } else if (value is sass.SassNumber) {
+      var number = Value_Number()..value = value.value * 1.0;
+      number.numerators.addAll(value.numeratorUnits);
+      number.denominators.addAll(value.denominatorUnits);
+      result.number = number;
+    } else if (value is sass.SassColor) {
+      // TODO(nweiz): If the color is represented as HSL internally, this coerces
+      // it to RGB. Is it worth providing some visibility into its internal
+      // representation so we can serialize without converting?
```
Comment (Awjin) [inline lib/src/protofier.dart:56]: I think this would be worth it. Would we also need to support HSL colors in the protocol?

**Review by Awjin — APPROVED** [review]

File: lib/src/protofier.dart (hunk 1 above)
Comment (nex3) [reply lib/src/protofier.dart:56]: Yeah, that would be the implication. At that point we'd probably *also* need to support HWB. Given how much overhead is going to be taken up serializing and deserializing, I'm not sure it's worth it...

Comment (johnfairh) [reply lib/src/protofier.dart:56]: I'd prefer the protocol to either (a) guarantee transparency on colour representation or (b) explicitly use some canonical rep [rgb?].  I have no idea about practical user-visible consequences of round-tripping hsl/rgb/hsl though.

eg. in my host Color implementation I added the 'preferred rep' method to know whether to gen `Value_RgbColor` or `Value_HslColor` but this is slightly pointless if the embedded side sometimes adds conversions: either the conversion is bad, so we shouldn't do it; or it's fine and so we may as well simplify current & future code (eg. hwb support) by doing it always and delete `Value_HslColor`.

(It feels to me that latency/cpu cost of doing additional conversions would be negligible compared to the cost of the whole protobuf IPC.)

Comment (nex3) [reply lib/src/protofier.dart:56]: Awjin and I discussed this and agreed that, to match behavior with native Sass functions, the compiler should probably pass HSL and HWB colors as-is to the embedded host. I'll make that happen in a followup CL.

**Pull request merged**